api/envoy/extensions/filters/http/local_ratelimit/v3/local_rate_limit.proto

```diff
@@ -19,8 +19,17 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // Local Rate limit :ref:`configuration overview <config_http_filters_local_rate_limit>`.
 // [#extension: envoy.filters.http.local_ratelimit]
 
-// [#next-free-field: 12]
+// [#next-free-field: 13]
 message LocalRateLimit {
+  // Defines the version of the standard to use for X-RateLimit headers.
+  enum XRateLimitHeadersRFCVersion {
```

Comment:
How is this enum different than: https://github.com/envoyproxy/envoy/blob/1aba2de82c714e8c7573c4efc997cc112eba4acf/api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto#L28-L34 ?
I think we should use the other enum, and not redefine a new one as this can cause a divergence in the future (e.g., global RL supporting draft 03, while local RL supporting draft 04). We should make this consistent.

Reply:
`DRAFT_VERSION_03` here means the responding x-ratelimit-* headers are according to draft RFC version 03. I think we should keep its name. And do you think it is a good idea for us to extract `XRateLimitHeadersRFCVersion` in `LocalRateLimit` and `RateLimit` and relocate it in common/ratelimit/v3/ratelimit.proto?

Reply:
I don't mind the name `DRAFT_VERSION_03`.

Yes, I do think it would be better to extract the enum to the extension's common ratelimit proto.

Reply:
An API change has been committed, but the CI reports the change is non-backwards-compatible. Is this a issue that needs to be fixed?